README.md

```diff
@@ -60,8 +60,3 @@ adding a new message or simply accepting that pylint is supposed to warn.
 the `messages` directory contains a list of files which should match the
 name of the test and contain error type, line number, class and expected text.
 These are useful to quickly add "expected messages".
-
-
-# License
-
-`pylint-django` is available under the GPLv2 license.
```

Comment:
Why have you deleted the license section? Please restore it back.

Reply:
This is unneeded and untypical. There a a dedicated LICENSE file present in this repository. GitHub automatically evaluates that.

Reply:
The original reason for that was because it's easier for a human reader to parse a single line of text than a whole LICENCE file - and this code won't always be distributed via GitHub. I mean, that's the point of the GPLv2! So relying on GitHub to help people understand it is not a good thing I would say.